frontend/src/Models.ts

```diff
@@ -0,0 +1,12 @@
+const MODELS = [
+  "gpt-4",
+  "gpt-4-0613",
+  "gpt-4-32k",
+  "gpt-4-32k-0613",
+  "gpt-3.5-turbo",
+  "gpt-3.5-turbo-0613",
+  "gpt-3.5-turbo-16k",
+  "gpt-3.5-turbo-16k-0613",
+];
```

Comment:
Now that we've got TypeScript, we could make this an enum, and replace any of these strings in the frontend with the enum value. e.g. `MODELS.OPENAI_GPT_4`.